sentences/it/_common.yaml

```diff
@@ -53,7 +53,7 @@ expansion_rules:
   to: "(a | al | allo | alla | agli | alle)"
   name: "{name}"
   area: "{area}"
-  what_is: "(qual è | quale è | quanto è | quant'è | come è | com'è)"
+  what_is: "(qual è | quale è | quanto è | quant'è | come è | com'è | che)"
```

Comment:
Please in a new PR translate the expansion rules to Italian unless they wrap a slot (ie `{name}`). So change `what_is` to `quanto` or something.

Reply:
Of course is possible but imho is better use en words to slots because, as you can see, there are too many words for "what_is" and they are used in different context. "what_is" btw wrap all of them and I don't think that a truly equivalent exists. Also if translated anyway, we (I) have to work with common.yaml opened as a dictionary.